controllers/volumereplicationgroup_controller.go

```diff
@@ -1038,8 +1038,8 @@ func (v *VRGInstance) updateVRGStatus(result ctrl.Result) ctrl.Result {
 	if !reflect.DeepEqual(v.savedInstanceStatus, v.instance.Status) {
 		v.instance.Status.LastUpdateTime = metav1.Now()
 		if err := v.reconciler.Status().Update(v.ctx, v.instance); err != nil {
-			v.log.Info(fmt.Sprintf("Failed to update VRG status (%v/%s)",
-				err, v.instance.Name))
+			v.log.Info(fmt.Sprintf("Failed to update VRG status (%v/%s/%s)",
```

Comment:
The v.log already logs the VRG namespaced name as part of its [initialization](https://github.com/RamenDR/ramen/blob/34891bf43bad0ff27262a3fbf4db2356e3e189fd/controllers/volumereplicationgroup_controller.go#L382), why the added repeat of the information here?